modules/document_repository/test/TestPlan.md

```diff
@@ -28,20 +30,11 @@
    [Manual Testing]
 14. Edit a file in the repository: check that “Date Uploaded” date is updated.
     [Manual Testing]
-15. Check that the preference “Receive Document Repository Notifications” works.
-    [Manual Testing]
-16. Check that the performance is OK with large number of files.
```

Comment:
Removing this line because it's unusual; we don't ask for stress testing in other modules and with file uploads it's especially difficult/unrealistic to do this in manual tests.

Reply:
Should it be tested automatically by a script?

Reply:
Should this still be tested but by an automated script?

Reply:
It sounds like a good idea but I don't think we have anything like that in our testing code.